docs/data/charts/funnel/funnel.md

```diff
@@ -67,6 +67,21 @@ It accepts a number that represents the gap in pixels.
 
 {{"demo": "FunnelGap.js"}}
 
+### Border radius
+
+The border radius of the sections can be customized by the `borderRadius` property.
+It accepts a number that represents the radius in pixels.
+
+- The `bump` curve interpolation will not respect the border radius.
+- The `linear` curve will respect the border radius to some extent due to the angle of the sections.
```

Comment:
```suggestion
- The `linear` curve respects the border radius to some extent due to the angle of the sections.
```

Can we be more specific of the problems? To which extent isn't the radius respected, or why isn't it respected?